packages/zosfiles/src/cli/upload/dtp/DirToPds.handler.ts

```diff
@@ -29,12 +29,18 @@ export default class DirToPdsHandler extends ZosFilesBaseHandler {
             stageName: TaskStage.IN_PROGRESS
         };
         commandParameters.response.progress.startBar({task: status});
-        const response = await Upload.dirToPds(session, commandParameters.arguments.inputdir, commandParameters.arguments.dataSetName, {
-            volume: commandParameters.arguments.volumeSerial,
-            binary: commandParameters.arguments.binary,
-            task: status,
-            responseTimeout: commandParameters.arguments.responseTimeout
-        });
+        const response = await Upload.dirToPds(
```

Comment:
I don't mean to be annoying, but could you maybe add a test to the dirToPds API related to the encoding option?
I understand that these lines are already covered by other tests, hence why overall coverage is not going down as much.

Reply:
As you mention encoding tests were covered in other parts, but I created a unit test for dirToPds API encoding from what I understood. I am not sure whether this test is redundant or serves the purpose. Any comments? @zFernand0

Reply:
Thank you very much for adding it!
Now, In case the option name changes in the future (or is removed), it will be more than just snapshots that fail, (which people sometimes update with the `jest -u` command) 😋 
It'd be nice to capture the behavior in a system test as well, but it will end up failing depending on the system configuration.